f/fcs/fn_fcsInit.sqf

```diff
@@ -0,0 +1,118 @@
+// F3 - FCS/Commander's Override system
+// Credits and documentation: https://github.com/folkarps/F3/wiki
+// Originally from: https://github.com/NikkoJT/njt_fcs (integrated with permission)
+
+/*
+	This component adds three main functions to a vehicle it's called on:
+	1. Commander's Override
+		The Commander's Override allows the commander to automatically point the gun at whatever object they're looking at.
+	2. FCS failure simulation
+		When the vehicle is hit by a high-calibre weapon, there is a small chance of an FCS failure, disabling the TC override, laser rangefinder, and NVDs, and force-setting the gunner's zeroing.
+		A unit with the repair trait can reset the FCS from the gunner's seat.
+		(If and when BI adds the ability to disable turret stabilisation by script, it will do that too)
+	3. Driver's brake release
+		The driver can override the vehicle's automatic brakes, allowing it to roll freely at low speed and on slopes. Braking re-engages the autobrakes.
+		
+	Execution, all clients:
+	[_vehicle] call f_fnc_fcsInit;
+	
+	Or example (warning: classnames are case sensitive):
+	_fcsvehicles = vehicles select {typeOf _x in ["I_APC_Wheeled_03_cannon_F","I_APC_tracked_03_cannon_F"]};
+	{[_x] call f_fnc_fcsInit} forEach _fcsvehicles;
+	
+	An option to activate this component is pre-placed in the F3 init.sqf
+
+*/
+
+// Doesn't need to run on DS
+if (isDedicated) exitWith {};
```

Comment:
This might be better as `not hasInterface`since `isDedicated` will run on a HC as well

Reply:
Done